### DISCUSSION TITLE
[Feature] Multi page TIF support

### DISCUSSION THREAD

**Discussion by CorentinB** [immich-app/immich#9243] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi, I really hope it's not a duplicate but I didn't find a proposition for this feature.

I usually scan photos in TIF files, with sometimes multiple pages in the TIF file, for example if the photo has handwritten text at the back. 

Right now, when importing such photo in my immich instance, only the first image is visible. 

Thanks!

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by Giterd** [comment]
Yes please!  I am archiving photo prints, most of which have info written on the back.  The recommended archiving format is TIFF, with each photo scanned into a file with two pages, front and back.  But in Immich only the front can be viewed, which means the associated metadata are not available.

**Comment by steve-burke** [comment]
+1 from me too.  Some of my albums contain a pdf "diary" related to the pictures in the album.  If I converted the pdf's to multi-page TIF's I could view the diary from within immich.

**Comment by mirisbowring** [comment]
An multipage TIFF could autocreate a Stack like every page beomes a photo within this "tiff stack" or so
